.tekton/images-mirror-set.yaml

```diff
@@ -40,6 +40,14 @@ spec:
         - quay.io/redhat-user-workloads/ose-osc-tenant/osc-dm-verity-image
         - quay.io/redhat-user-workloads/ose-osc-tenant/osc-dm-verity-image-v1-10
       source: registry.redhat.io/openshift-sandboxed-containers/osc-dm-verity-image
+    - mirrors:
+        - quay.io/redhat-user-workloads/ose-osc-tenant/osc-pccs
+        - quay.io/redhat-user-workloads/ose-osc-tenant/osc-pccs
```

Comment:
Please drop the duplicate lines here and there.

The existing mirroring rules have two entries, one for the devel branch and one for OSC 1.10 (not applicable to these new images you're adding).